components/web-apps/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
+  - elikatsis
+  - kimwnasptd
+  - StefanoFioravanzo
+reviewers:
+  - avdaredevil
```

Comment:
Have @avdaredevil and @prodonjs agreed to participate in this? 

Do not duplicate reviewers and approvers. Please refer to
https://www.kubeflow.org/docs/about/contributing/#maintaining-owners-files

Reply:
My bad, I just copied the reviewers from the jupyter web app.
Lets start with an empty list of reviewers for now